<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>container</class>
 <widget class="QWidget" name="container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>1141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="write_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>650</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>WRITE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="read_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>650</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>READ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>650</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain_txt">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>650</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>630</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug Output</string>
   </property>
  </widget>
  <widget class="QListWidget" name="iso_list_widget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>401</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="iso_list_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ISO LIST</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DISK IMAGE MANAGER APP</string>
   </property>
  </widget>
  <widget class="QLabel" name="message_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>550</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="action_tips_label">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>720</y>
     <width>1111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iso_tips_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleziona la ISO da scrivere sul dispositivo</string>
   </property>
  </widget>
  <widget class="QLabel" name="device_tips_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleziona la ISO da scrivere sul dispositivo</string>
   </property>
  </widget>
  <widget class="QListWidget" name="device_list_widget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>401</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="device_list_label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DEVICES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>REFRESH DEVICES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_iso_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE ISO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
